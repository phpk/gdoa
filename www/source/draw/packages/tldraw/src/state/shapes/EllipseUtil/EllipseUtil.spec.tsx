import { Ellipse } from '..'

describe('Ellipse shape', () => {
  it('Creates a shape', () => {
    expect(Ellipse.create({ id: 'ellipse' })).toMatchSnapshot('ellipse')
  })
})
