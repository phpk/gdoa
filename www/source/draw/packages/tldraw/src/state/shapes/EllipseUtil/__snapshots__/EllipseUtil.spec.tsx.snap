// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ellipse shape Creates a shape: ellipse 1`] = `
Object {
  "childIndex": 1,
  "id": "ellipse",
  "name": "Ellipse",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "radius": Array [
    1,
    1,
  ],
  "rotation": 0,
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "ellipse",
}
`;
